resources/docs/cache-in-ci.md

```diff
@@ -39,8 +39,8 @@ On GitHub Actions, you can use the [built-in cache action](https://github.com/ac
         uses: actions/cache@v3
         with:
           path: ./var/cache/rector
-          key: ${{ runner.os }}-rector-${{ hashFiles('**/composer.lock') }}
-          restore-keys: ${{ runner.os }}-rector-
+          key: ${{ steps.setup-php.outputs.php-version }}-rector-${{ github.run_id }}
```

Comment:
```suggestion
          key: ${{ runner.os }}-rector-${{ hashFiles('**/composer.lock') }}-${{ github.run_id }}
```

Hi @staabm !

* `${{ runner.os }}` still seems necessary, unless we set `enableCrossOsArchive`
* `${{ steps.setup-php.outputs.php-version }}` I do not have this step or output in my setups. So it would be confusing if the docs just assume its existence. 
* **/composer.lock seems neccessary to me, as it hashes the state of vendor php classes as well as the rector version.

Reply:
is that matter committed or not the composer.lock?

Reply:
The `hashFiles('**/composer.lock')` only makes sense if the `composer.lock` is committed. If `composer.lock` does not exist, the hash function will return an empty string (no harm done).

Reply:
@staabm could you verify it and then we can merge it ?

Reply:
its a matter of taste. IMO `setup-php` is used in any project I know which uses github action.
we don't use `${{ hashFiles('**/composer.lock') }}` in our projects and did not have problems .. therefore I would not use it and keep it simpler

Reply:
no matter how we turn the example mentioned in the docs, its pretty unlikely that in a project it can be used 1:1